src/main/kotlin/no/nav/familie/tilbake/behandling/FagsakService.kt

```diff
@@ -42,4 +42,18 @@ class FagsakService(val fagsakRepository: FagsakRepository,
         }
         return FinnesBehandlingsresponsDto(finnesÅpenBehandling = finneÅpenBehandling)
     }
+
+    @Transactional(readOnly = true)
+    fun hentBehandlingerForFagsak(fagsystem: Fagsystem,
```

Comment:
kunne vurdert å gjøre henting i 1 spørring ala `select * from behandling where fagsakId in (select fagsakid from fagsak where eksternfagsakid = ? )`

Men dette fungerer bra fremt til det blir veeeldig mye trafikk :)